Workbooks/Virtual Machines - Network Dependencies/Connections Overview - Single VM/Connections Overview.workbook

```diff
@@ -727,7 +682,77 @@
             "treeType": 0,
             "expanderColumn": "Name",
             "expandTopLevel": true
-          }
+          },
+          "labelSettings": [
+            {
+              "columnId": "ComputerName",
+              "label": "ComputerName"
```

Comment:
did you explicitly add these?  if so, the labels should be nice names, with spaces and things, like "Computer Name".  right now, they're all the same as the keys and this is probably a bug @ericajlin ?

Reply:
Hi John, I did not explicitly add these. The template itself is 6 month old so lots of new metadata is added to the template by default. What I typically do is load the template, make changes, and manually remove any hard-coded entries and update the PR. I will go back and update some of these labels.